shared/tab-bar/index.render.desktop.js

```diff
@@ -1,132 +1,110 @@
-  ...(flags.tabChatEnabled ? {[chatTab]: 'iconfont-chat'} : null),
-  [folderTab]: 'iconfont-folder',
-  [devicesTab]: 'iconfont-device',
-  [settingsTab]: 'iconfont-settings',
-}
-
-export type SearchButton = 'TabBar:searchButton'
-export const searchButton = 'TabBar:searchButton'
-
-function tabToIcon (t: Tab): IconType {
-  return icons[t]
-}
-
-export default class TabBarRender extends Component<void, Props, void> {
-  _renderSearch (onClick: () => void) {
-    const source = {type: 'nav', icon: 'iconfont-nav-search'}
-    const button = (
+export default function TabBar ({selectedTab, onTabClick, username, badgeNumbers}: Props) {
+  const avatar = (
+    <Avatar
+      size={32}
+      onClick={() => onTabClick(profileTab)}
+      username={username}
+      borderColor={selectedTab === profileTab ? globalColors.white : globalColors.blue3_40}
+    />
+  )
+
+  return (
+    <Box style={stylesTabBar}>
       <TabBarButton
         label='Search'
-        selected={this.props.searchActive}
-        source={source} />
-    )
-
-    return (
-      <TabBarItem
-        key='search'
-        tabBarButton={button}
-        selected={!!this.props.searchActive}
-        onClick={onClick}
-        style={{...stylesTabBarItem}}
-      >
-        <Box style={{flex: 1, ...globalStyles.flexBoxColumn}}>{this.props.searchContent || <Box />}</Box>
-      </TabBarItem>
-    )
-  }
-
-  _renderProfileButton (tab: Tab, selected: boolean, onClick: () => void) {
-    // $FlowIssue
-    const avatar: Avatar = <Avatar size={32} onClick={onClick} username={this.props.username} borderColor={selected ? globalColors.white : globalColors.blue3_40} />
-    const source = {type: 'avatar', avatar}
-    const label = this.props.username
-    return (
+        selected={selectedTab === searchTab}
+        onClick={() => onTabClick(searchTab)}
+        source={{type: 'nav', icon: 'iconfont-nav-search'}}
+        style={stylesTabButton}
+      />
       <TabBarButton
-        label={label}
-        selected={selected}
-        badgeNumber={this.props.badgeNumbers[tab]}
-        source={source} />
-    )
-  }
-
-  _renderNormalButton (tab: Tab, selected: boolean, onClick: () => void) {
-    const source = {type: 'nav', icon: tabToIcon(tab)}
-    const label = tabPrettify(tab)
-    return (
+        label='Folders'
+        selected={selectedTab === folderTab}
+        onClick={() => onTabClick(folderTab)}
+        badgeNumber={badgeNumbers[folderTab]}
+        source={{type: 'nav', icon: 'iconfont-folder'}}
+        style={stylesTabButton}
+      />
+      {flags.tabChatEnabled &&
+        <TabBarButton
+          label='Chat'
+          selected={selectedTab === chatTab}
+          onClick={() => onTabClick(chatTab)}
+          badgeNumber={badgeNumbers[chatTab]}
+          source={{type: 'nav', icon: 'iconfont-chat'}}
+          style={stylesTabButton}
+        />
+      }
+      {flags.tabPeopleEnabled &&
+        <TabBarButton
+          label='Chat'
```

Comment:
Oops! This doesn't make sense.  It's probably a rebase or copy-paste mistake from updating these tabs. Will fix. :sweat: 

I think the people tab on native currently shows the search UI, and native doesn't have a search tab. At least, that's what I gathered from the previous code. The mobile designs appear to be updated and now match desktop, but we haven't implemented it yet.